README.md

```diff
@@ -46,8 +46,8 @@ A library that enables the reuse of [MDC-Android][mdc] Material 2 XML themes, fo
 ### 🎨 [Material 3 Theme Adapter](./themeadapter-material3/)
 A library that enables the reuse of [MDC-Android][mdc] Material 3 XML themes, for theming in Jetpack Compose.
 
-### 📖 [Pager](./pager/) (Soon to be deprecated - Upstreamed in Compose 1.4)
-A library that provides utilities for building paginated layouts in Jetpack Compose, similar to Android's [ViewPager][viewpager].
+### 📖 [Pager](./pager/) (Deprecated)
```

Comment:
Please move to bottom of the readme with the other deprecated libraries

Reply:
Done